### DISCUSSION TITLE
Bulk load a new graph using CQLOutputFormat

### DISCUSSION THREAD

**Discussion by cdegroc** [JanusGraph/janusgraph#2682] [Q&A]
Hi 👋 

I'm using the `CQLInputFormat` to dump data from JanusGraph using MapReduce/Spark.
I've noticed JanusGraph does not provide a `CQLOutputFormat` to do the opposite operation (load data).
I was wondering if the community had experience with Cassandra's [CQLOutputFormat](https://github.com/apache/cassandra/blob/trunk/src/java/org/apache/cassandra/hadoop/cql3/CqlOutputFormat.java) to bulk load a new graph.
Unless I am missing something, I guess a bulk load + a reindex could maybe be faster than using the SparkGraphComputer and OLAP traversals?

Any feedback appreciated. Thanks!

**Comment by li-boxuan** [comment]
I have no relevant experience but it sounds interesting! Maybe it is worth starting a discussion on the TinkerPop user mailing list too. I presume this is not specific to JanusGraph.
